plugins/PrivacyManager/PrivacyManager.php

```diff
@@ -351,7 +351,8 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'PrivacyManager_GeolocationAnonymizeIpNote';
         $translationKeys[] = 'PrivacyManager_GetPurgeEstimate';
         $translationKeys[] = 'PrivacyManager_HowDoIAskForConsent';
-        $translationKeys[] = 'PrivacyManager_HowDoIAskForConsentIntroduction';
```

Comment:
Obsolete translation keys need to be removed from `en.json`. Weblate will then automatically remove it from the other translation files when they are updated.
In general we should always prefer adding / removing translations instead of changing existing ones, unless it's only fixing grammar or typos. The translations on weblate aren't removed when a translation source is changed and only marked as `needs editing`. This means that in this case incorrect/outdated existing translations might be displayed until someone updated them.